### DISCUSSION TITLE
NUnitLite does not notify teamcity about failed tests on OneTimeSetup failure

### DISCUSSION THREAD

**Discussion by zelins** [nunit/nunit#4161] [Q&A]
Seems that there is a bug in NUnitLite runner (AutoRun class)

On our project we currently have 25 tests, 14 of them are dependent on OneTimeSetUp method.
When OneTimeSetup method throws an unhandled exception then 14 tests are just skipped and team city step passes successfully despite of these 14 failed tests. It just shows that 11 tests passed

build 307 threw the exception in OneTimeSetUp
build 308 didn't throw anything
![image](https://user-images.githubusercontent.com/10038169/186893556-3aafce84-57c4-4433-bf4f-8bafc7c7299b.png)

**Comment by zelins** [comment]
I forgot to note that we use NUnitLite version 3.12.0 but it seems that the problem is still relevant for the latest stable version

**Comment by rprouse** [comment]
What is the behavior when running from the command line? Does NUnitLite properly fail/error your tests?
